### ISSUE TITLE
Pyglow not loading in Travis

### ISSUE THREAD

**Issue by jklenzing** [rstoneback/pysat#154]
Currently, Travis CI gives an error when pyglow is loaded when testing the sgp4 files. 
```
ModuleNotFoundError: No module named 'pyglow'
```

**Comment by jklenzing** [comment]
`pyglow_install.sh` no longer the default method to install `pyglow`

**Comment by jklenzing** [comment]
Note for future edits: Using

```
python setup.py install --user
```

instead of

```
python setup.py install
```

will result in incorrect travis install, and `pyglow` cannot be imported from within `pysat`.

**Comment by jklenzing** [comment]
Closed with merge of #155.

**Comment by rstoneback** [comment]
Is there any way we can engage the --user option from the setup.py file, or something like that? I'm guessing most people aren't going to think to include the --user flag when installing pysat.

Normally Fridays are my review day but this Friday is covered up by Faculty meetings, performance reviews, proposals, and more! :)

**Comment by jklenzing** [comment]
We shouldn't need the --user flag.  It's just that whatever's going on in the travis environment seemed to get annoyed if I installed pyglow with the --user option, so I removed it for pyglow.  Granted, the fact that #143 passed travis before this was merged in, so it's *possible* the issue was something they fixed up the line over the weekend.  It's ***entirely possible*** that they fixed it at the same time I implemented this change, and it has nothing to do with the changes in my code.  :)

Right now, pyglow includes --user as part of the standard install, pysat does not.  This is how my computer is set up, so I'll play with some local tests now that python 3 can run the pyglow / sgp4 code to make sure that --user doesn't affect it.
